source/documentation/data-docs/curated-databases-docs/databases-for-analysis-and-apps.md

```diff
@@ -0,0 +1,58 @@
+
+- Apps cannot access (read/write) S3 buckets created by the AP users (instead they have their own buckets prefixed with `alpha-app-`)
+- Apps cannot access Databases created for the AP users (these are all of the curated databases created by the Data Engineers)
+- Users can access read/write to App buckets (permissions to do this are given via the Control Panel).
+
+So if you create an app that needs specific data from our databases then you need to consider the following practices:
+
+#### Data Minimisation
+
+When writing data to your app-bucket you should only ever give the app what it needs. I.e. even though your analysis might require personal or low level data. If the calculations on your App do not need the data at this lower level, then only write data to the app-bucket that is an aggregate of the data you take. You should never put data in the app-bucket that you do not want users to see (even if the code behind the app itself has no method to expose that data to the user via the app).
+
+
+#### Writing data to an App bucket
+
+There are multiple ways you can do this and it is up to you to determine how best to do it.
+
+- You could manually run a script from your AP User instance that writes the necessary data to the app bucket whenever it needs updating
+- You could write an airflow script that writes data from a database to the app bucket (this is because the airflow role will be seperate to the app role). Meaning the app still only has access to it's very specific area in AWS.
+- If you want a database backend but nothing as big as Athena you could write your data into an SQLite and save to to the app-bucket. Then just like how R/Python would read in a file from S3. You could load in SQLite database into memory and use that as a backend database for your app.
+
+> When updating data for Rshiny Apps you often need to restart the R session so it refreshes the cached data. You can do this using airflow (to see how to do this search the airflow-dags repository for the `BashOperator`).
+
+
+#### Creating a database for your app
+
+> Note this is only recommended for users who understand how to create/manage databases on Athena. See other parts of guidance on these tools for more details on how to use them.
+
+In most cases you will not need to do this and it is unnecessary. However, if you feel like you do need to create an Athena SQL database because your app requires the processing power from Athena and the data you are reading in is too large to be cached and there is no way around it then you will need to do the following:
```

Comment:
You've got two 'and's here: processing power and too large and no way around it. Either rewrite it, or take one of the things out, or replace the first 'and' with a comma.